src/utils/base64.h

```diff
@@ -0,0 +1,59 @@
+#include <string>
+
+#include <torrent/exceptions.h>
+
+namespace utils {
+constexpr static char padCharacter = '=';
+static std::string
+base64decode(const std::string& input) {
```

Comment:
Where is this from?

Reply:
I originally stole it from  https://github.com/jesec/rtorrent/blob/199e8f85244c8eb1c30163d51755570ad86139bb/src/core/manager.cc#L46, but it looks to be a modified version of https://en.wikibooks.org/wiki/Algorithm_Implementation/Miscellaneous/Base64#C++_2

Reply:
Add the reference and that it is public domain as a comment.

Rename `padCharacter` to `base64_pad_character`.

Remove 'static' from the function definition.

Also, add newlines between functions, namespaces, etc.